gulpfile.js

```diff
@@ -14,6 +14,7 @@ const responsive = require('gulp-responsive');
 const merge = require('merge-stream');
 const postcss = require('gulp-postcss');
 const autoprefixer = require('autoprefixer');
+const inlinesource = require('gulp-inline-source');
```

Comment:
I think we should move to https://github.com/exuanbo/gulp-inline-source since https://github.com/fmal/gulp-inline-source looks to be no more maintained for few years now

Reply:
I browsed the most used plugin in `gulp` and found this version. Did not look at its maintenance status. Will change to actively maintained version.